Assets/Scripts/GroundPound.cs

```diff
@@ -79,7 +68,7 @@ private IEnumerator DropAndSmash()
     public void CompleteGroundPound()
```

Comment:
Right now this is called every time we hit the ground or a wall, can you add an if statement so that is only runs on completions of a groundpound

Reply:
Would getting rid of the if-statement on lines 42-45 work?:
if (!Player.isGrounded)
        {
            doGroundPound = true;
        }

Reply:
I would just have a simple bool that keeps track if I am currently in a ground pound. Set it to true when the ground pound is started, and set it to false when the ground pound ends. You only need to run that function if that bool is true.

Reply:
Where would this if-statement be? In Update? Or in the Started function?

Reply:
I would put the if statement before function call of CompleteGroundPound, like in OnCollisionEnter2D